package.json

```
@@ -145,7 +145,7 @@
     "react-split-pane@^0.1.92": "patch:react-split-pane@npm%3A0.1.92#./.yarn/patches/react-split-pane-npm-0.1.92-93dbf51dff.patch",
     "tmp": "0.2.1",
     "tsx": "^4.7.2",
-    "typescript": "5.7.2"
+    "typescript": "~5.8.0"
```

I don't think this changes anything practically, but we've gone with explicit numbers before and I don't know of a good reason to change it:

```suggestion
    "typescript": "5.8.2"
```

Mild preference from me on going explicit just to make things... well, explicit.

---

I assume that's what the lock is for, right? Anyway, used a fixed version instead 😉